dev/all_tests.yaml

```diff
@@ -340,4 +340,4 @@ tests:
     api: gate
     args:
       stage_name: randomWait
-      stage_params: '{ "maxWaitTime": 5 }'
+      stage_params: '\"{ "maxWaitTime": 5 }\"'
```

Comment:
I think this still won't work, as now bash will see `"{ "maxWaitTime": 5 }"` and match up the quotes incorrectly.  I think the inner ones should stay `'` but need to be escaped.

Reply:
Yes... unfortunately the YAML parser sees the first three characters of`'\'{ "maxWaitTime": 5 }\''`, thinks I just want `'\'`, and then gets confused when it sees `{`.

Reply:
(But you're right)